src/test/java/lotto/domain/result/LottoStatisticsTest.java

```diff
@@ -0,0 +1,77 @@
+package lotto.domain.result;
+
+import lotto.domain.number.LottoNumber;
+import lotto.domain.ticket.LottoTicket;
+import lotto.domain.ticket.LottoTickets;
+import lotto.domain.ticket.WinningLotto;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LottoStatisticsTest {
+
+    public static final int LOTTO_TICKET_PRICE = 1000;
+    private LottoTickets lottoTickets;
+    private WinningLotto winningLotto;
+
+    @BeforeEach
+    void setUp() {
+        LottoTicket ticket1 = LottoTicket.of(
+                List.of(
+                        LottoNumber.of(1), LottoNumber.of(2),
+                        LottoNumber.of(3), LottoNumber.of(4),
+                        LottoNumber.of(5), LottoNumber.of(6)
+                )
+        );
```

Comment:
로또 티켓 1장을 생성할 때 각각의 번호를 `LottoNumber`로 감싸서 넘겨줘야 하는 게 다소 번거로울 것 같아요.
테스트 픽스처를 만들어두거나, 아니면 보다 쉽게 생성할 수 있는 편의 메서드를 제거해보는 건 어떨까요?
```suggestion
        LottoTicket ticket1 = LottoTicket.of(List.of(1, 2, 3, 4, 5, 6));
```

Reply:
네 반영하겠습니다.

Reply:
LottoNumber를 생성하는 테스트용 클래스를 따로 만들어서 적용해봤습니다.